js/poloniexfutures.js

```diff
@@ -0,0 +1,1599 @@
+                'spot': false,
+                'margin': true,
+                'swap': true,
+                'future': false,
+                'option': undefined,
+                'createOrder': true,
+                'fetchBalance': true,
+                'fetchClosedOrders': true,
+                'fetchCurrencies': false,
+                'fetchFundingRate': true,
+                'fetchL3OrderBook': true,
+                'fetchMarkets': true,
+                'fetchMyTrades': true,
+                'fetchOHLCV': true,
+                'fetchOpenOrders': true,
+                'fetchOrder': true,
+                'fetchOrderBook': true,
+                'fetchOrdersByStatus': true,
+                'fetchPositions': true,
+                'fetchTicker': true,
+                'fetchTickers': true,
+                'fetchTime': true,
+                'fetchTrades': true,
+            },
+            'timeframes': {
+                '1m': 1,
+                '5m': 5,
+                '15m': 15,
+                '30m': 30,
+                '1h': 60,
+                '2h': 120,
+                '4h': 480,
+                '12h': 720,
+                '1d': 1440,
+                '1w': 10080,
+            },
+            'urls': {
+                'logo': 'https://user-images.githubusercontent.com/1294454/27766817-e9456312-5ee6-11e7-9b3c-b628ca5626a5.jpg',
+                'api': {
+                    'public': 'https://futures-api.poloniex.com',
+                    'private': 'https://futures-api.poloniex.com',
+                },
+                'www': 'https://www.poloniex.com',
+                'doc': 'https://futures-docs.poloniex.com',
+                'fees': 'https://poloniex.com/fee-schedule',
+                'referral': 'https://poloniex.com/signup?c=UBFZJRPJ',
+            },
+            'api': {
+                'public': {
+                    'get': {
+                        'contracts/active': 1,
+                        'contracts/{symbol}': 1,
+                        'ticker': 1,
+                        'tickers': 1, // v2
+                        'level2/snapshot': 0.05,
+                        'level2/depth': 0.05,
+                        'level2/message/query': 0.05,
+                        'level3/snapshot': 0.05, // v2
+                        'trade/history': 1,
+                        'interest/query': 1,
+                        'index/query': 1,
+                        'mark-price/{symbol}/current': 1,
+                        'premium/query': 1,
+                        'funding-rate/{symbol}/current': 1,
+                        'timestamp': 1,
+                        'status': 1,
+                        'kline/query': 1,
+                    },
+                    'post': {
+                        'bullet-public': 1,
+                    },
+                },
+                'private': {
+                    'get': {
+                        'account-overview': 10,
```

Comment:
Aren't we missing `marginMode` endpoints?
https://futures-docs.poloniex.com/#query-margin-mode
https://futures-docs.poloniex.com/#change-margin-mode

The latter can be used for `setMarginMode`

Reply:
@carlosmiei Is there an example for `fetchMarginMode`? No results on a global search with as input: `'fetchMarginMode': true`